<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchesInSquads</class>
 <widget class="QWidget" name="MatchesInSquads">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Матчи состава</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_Exit">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_Squads"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_AddSquad">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_DeleteSquad">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_AllMatches">
           <property name="text">
            <string>Все матчи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_PlayersInSquads">
           <property name="text">
            <string>Игроки состава</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_CoachStaff">
           <property name="text">
            <string>Тренеры состава</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tw_Matches"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ChangeMatch">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
